tools/constants.py

```diff
@@ -45,6 +45,9 @@
 SUBDIVISION_NAME = "subdivision_name"
 MUNICIPALITY = "municipality"
 BOUNDING_BOX = "bounding_box"
+STATUS = "status"
+ACTIVE = "active"
```

Comment:
Same thing for all status constants.
```suggestion
ACTIVE = "active"
DEPRECATED = "deprecated"
INACTIVE = "inactive"
DEVELOPMENT = "development"
```

Reply:
added.